### PR TITLE
added code example to get test plan variables.

### PR DESCRIPTION
**Pull request by myadav27**

### CODE REVIEW

File: azure_devops_rust_api/examples/test_plan.rs [hunk 1]
```diff
@@ -20,18 +20,46 @@ async fn main() -> Result<()> {
...
     println!("{:#?}", test_plans);
 
+    // Get all suites for a test plan for a project
+    // println!("The suites for the a plan for project are:");
+    // let test_suites = test_plan_client
+    //     .test_suites_client()
+    //     .get_test_suites_for_plan(&organization, &project, test_plan_id)
+    //     .await?
+    //     .value;
+    // println!("{:#?}", test_suites);
```
Comment (johnbatty) [inline azure_devops_rust_api/examples/test_plan.rs:44]: Why is this block commented out?

Comment (myadav27) [reply azure_devops_rust_api/examples/test_plan.rs:44]: This part of code is passing locally but failing in the pipeline , so commented. out to debug further

File: azure_devops_rust_api/examples/test_plan.rs [hunk 2]
```diff
@@ -20,18 +20,46 @@ async fn main() -> Result<()> {
...
     let project = env::var("ADO_PROJECT").expect("Must define ADO_PROJECT");
-
+    // let test_plan_id_value = env::var("TEST_PLAN_ID").expect("Must define PLAN_ID for the test");
+    // let test_plan_id:i32 = test_plan_id_value.parse().expect("Failed to parse TEST_PLAN_ID");
     // Create test_plan client
     let test_plan_client = test_plan::ClientBuilder::new(credential).build();
 
     // Get all test plans for project
-    println!("The test runs for project are:");
+    println!("The test plan for project are:");
```
[the remark this answers is not in the document]
Comment (myadav27) [reply azure_devops_rust_api/examples/test_plan.rs:31]: Correct Please refer [lint and typo fix](https://github.com/microsoft/azure-devops-rust-api/pull/126/commits/c8108466416f192b9953a1d80a0013f1c1cf3d71)

File: azure_devops_rust_api/examples/test_plan.rs (hunk 1 above)
Comment (myadav27) [reply azure_devops_rust_api/examples/test_plan.rs]: Correct Please refer [lint and typo fix](https://github.com/microsoft/azure-devops-rust-api/pull/126/commits/c8108466416f192b9953a1d80a0013f1c1cf3d71)

**Comment by johnbatty** [comment]
Thanks!

**Pull request merged**